app/components/Texting/components/FormView/FormView.jsx

```diff
@@ -55,13 +68,17 @@ const FormView = ({ service, handleSubmit, closeModal }) => {
           <div className={styles.dataRates}>
             *Standard text and data rates may apply.
           </div>
-          <input
-            type="text"
-            name="phoneNumber"
-            className={styles.input}
-            value={phoneNumber}
-            onChange={onChange}
-          />
+          <div className={styles.phoneInputBox}>
+            <span className={styles.phoneInputPrefix}>+1</span>
+            <input
+              type="text"
```

Comment:
I know that the previous version of this code didn't do this either, but could you change the type to [`tel`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel)? This is a special input element type that mostly looks like a normal text box, but on most mobile devices will cause a numeric keypad to pop open instead of a full text keyboard, which makes it easier to enter numbers.

Also, now that I'm looking at the docs for the `tel` input type, I'm seeing that it actually has a builtin `pattern` attribute for specifying a regex for validating the field, so maybe we should be using that instead of building our own custom JavaScript logic for it? Sorry for not remembering about that attribute beforehand 😅.